hw/ip/i2c/doc/_index.md

```diff
@@ -1,3 +1,223 @@
+
 ---
-title: "I2C Technical Specification" 
+title: "I2C HWIP Technical Specification"
 ---
+
+
+# Overview 
+
+This document specifies I2C hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.](https://github.com/lowRISC/stwg-base/blob/master/doc/ComportabilitySpecification.md)
+See that document for integration overview within the broader top level system.
+
+
+## Features 
+
+- Two pin clock-data parallel bidirectional external interface 
+- The initial revision only supports I2C as a Host (I2C Master).
```

Comment:
Please do the similar footnote as in the comportability specification:


```md
- the initial revision only supports I2C as a host ("I2C Master"<sup>1</sup>)

(below bulleted list)
<sup>1</sup>lowRISC is avoiding the fraught terms master/slave and defaulting to host/device where applicable.
```

Reply:
Thanks Scott.  Though changing your terminology here slightly in the footnote: Device->Target.
(overloading the word device causes problems in readability).